dll/win32/advapi32/advapi32.spec

```diff
@@ -432,13 +432,13 @@
 432 stdcall OpenServiceA(long str long)
 433 stdcall OpenServiceW(long wstr long)
 434 stdcall OpenThreadToken(long long long ptr)
-435 stub OpenTraceA
-436 stub OpenTraceW
+435 stdcall OpenTraceA(ptr)
+436 stdcall OpenTraceW(ptr)
```

Comment:
By looking at wine's advapi32.spec, I noticed they used "stdcall -ret64" because the returned TRACEHANDLE is a ULONG64 (everytime). Is it mandatory to have it there too? cc @ThFabba .

Reply:
Yeah that actually changes the calling convention.
In that case, wouldnt this be better?
`#define INVALID_PROCESSTRACE_HANDLE ((TRACEHANDLE)(-1))`

Reply:
I think that "to remain compatible with MS PSDK" we should keep "INVALID_HANDLE_VALUE" instead of the -1 ... ? Isn't the aim here to sign-extend? Then what about using LONG_PTR ?

Reply:
Yes, adding `-ret64` to our spec file also is a good idea. It's only used for relay tracing AFAIK, but since we have a similar feature it's good to keep the spec entry correct.

As for zero vs sign extension... see the note at https://msdn.microsoft.com/en-us/library/windows/desktop/aa364089(v=vs.85).aspx
So we should probably just never use that constant and return `(TRACEHANDLE)-1`. As for making the definition PSDK-compatible, we should `#define INVALID_PROCESSTRACE_HANDLE ((TRACEHANDLE)(ULONG_PTR)INVALID_HANDLE_VALUE` since with MS's compiler `printf("%I64x\n", (unsigned __int64)(void *)-1);` prints `ffffffff`.

Reply:
So how about I just change the code to return (TRACEHANDLE)-1; for this pr, drop the psdk commit and we have stubs?
Since that is the purpose of pr.

Reply:
Sure. Or you could even leave things as they are now.